src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -143,6 +248,10 @@ private static string GetClassificationForIdentifier(SyntaxToken token)
                 }
                 return ClassificationTypeNames.LocalName;
             }
+            else if (token.Parent is SingleVariableDesignationSyntax singleVariableDesignation && singleVariableDesignation.Identifier == token)
+            {
+                return ClassificationTypeNames.LocalName;
```

Comment:
might have to be careful with this one.  I'm not convinced that SingleVariableDesignationSyntax  won't be used in other locations in the future.

Reply:
i'm still concerned about this.

Reply:
How would you recommend future proofing this?

Reply:
enumerate all the known cases.